sky/serve/replica_managers.py

```diff
@@ -222,9 +270,13 @@ def is_scale_down_succeeded(self, initial_delay_seconds: int) -> bool:
         if self.sky_launch_status == ProcessStatus.INTERRUPTED:
             return True
         if self.sky_launch_status != ProcessStatus.SUCCEEDED:
-            return False
+            # sky_launch_status == RUNNING: a scale down happened before
+            # the sky.launch finished.
+            return self.sky_launch_status != ProcessStatus.FAILED
```

Comment:
Similar question as before, if the launch failed due to the capacity issue, should we consider it as scale_down_succeeded?

Reply:
Why should it be `scale_down_succeeded`?